app/(main)/register/page.test.tsx

```diff
@@ -43,6 +46,28 @@ jest.mock('../../../context/AuthContextProvider', () => ({
   },
 }));
 
+const TestButton = () => {
```

Comment:
This is incorreclty tested. Remove this.

Reply:
Removed